Makefile

```diff
@@ -388,6 +397,9 @@ tags-xnvme-public:
 	@$(CTAGS) --c-kinds=dfpgs -R include/lib*.h || true
 	@echo "## xNVMe: make tags-xnvme-public [DONE]"
 
+define _require_builddir-help
+# This is not intended to be invoke via cli, rather it is use by other targets
```

Comment:
> # This is not intended to be invoke via cli, rather it is use by other targets

# This is not intended to be invoked via the cli, rather it is used by other targets